.github/workflows/push-image.yml

```diff
@@ -29,17 +29,17 @@ jobs:
 
     - name: Create Builder Image
       run: |
-        pack create-builder builder-tmp --config builder.toml
+        pack create-builder builder --config builder.toml
 
     - name: Push
       env:
-        GCR_PUSH_BOT_JSON_KEY: ${{ secrets.GCR_PUSH_BOT_JSON_KEY }}
+        PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD }}
       run: |
-        echo "${GCR_PUSH_BOT_JSON_KEY}" | docker login --username _json_key --password-stdin gcr.io
-        docker tag builder-tmp "gcr.io/paketo-buildpacks/builder-tmp:full"
-        docker tag builder-tmp "gcr.io/paketo-buildpacks/builder-tmp:latest"
-        docker tag builder-tmp "gcr.io/paketo-buildpacks/builder-tmp:${{ steps.event.outputs.tag }}-full"
-
-        docker push "gcr.io/paketo-buildpacks/builder-tmp:full"
-        docker push "gcr.io/paketo-buildpacks/builder-tmp:latest"
-        docker push "gcr.io/paketo-buildpacks/builder-tmp:${{ steps.event.outputs.tag }}-full"
+        echo "${PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD}" | docker login --username paketobuildpacks --password-stdin
```

Comment:
I feel like its generally good practice to store the username in a secret as well... not that it is particularly secret in this case. Also, have we thought about converting `paketobuildpacks` from a community user to a community organization so we could then log in as a  member. https://docs.docker.com/docker-hub/orgs/

Reply:
I'm not sure if this is the best place for this discussion, but I'm curious what motivation there is for switching to a community organization? Is it just the ability to log in as a member?

Reply:
Yeah @ekcasey I hear you on that. I was sort of on the fence. I'll pop it in a secret for consistency sake. Wrt the transition to a community org, would that mean that we'd just end up having a new username and pass to use for this operation?

Reply:
You could still log in as the org itself, but the org can have members and you could also log in as a member. That way you could configure the permissions of both real and bot team members as appropriate. https://docs.docker.com/docker-hub/orgs/

No need to do it now, just thought I would bring mention it as a possibility for the future.